### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] Icon: Label not read

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11276]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Table with Filer:
The Back button of Filter dialog when we click Filter by is just mentioned as "toggle button' and label is not mentioned. Label for the UI must be mentioned.
#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with Filter on toolbar. click filter icon .to open there respective dialog popup. click filter by any attribute. The Back button visible will be just mentioned as "toggle button' and label is not mentioned.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?
Refer attachement :

<img width="638" alt="Screenshot 2024-01-21 at 4 29 47 PM" src="https://github.com/SAP/fundamental-ngx/assets/157268666/56818fca-dc16-4b85-aeb0-34117480b3cf">

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

The icon only back button needs to have a label according to Fiori: 
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/c10aea83-859b-42bb-bbf5-cbf97a3775c8)

Please after adding the label check if it is interpreted as accessibleName of the button. If not, you need to add the same tooltip text in the aria-label attribute.

Best Regards,
Nikolay

**Comment by JyotiKumari1507** [comment]
@droshev Please find the attached screen shot where The Back button is just mentioned as "**toggle button**' and label is not mentioned.. this filter dialog box is of platform table. we can't edit lable of back button
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/7e0fa504-90f8-42f4-afc7-69d84af3eeb9">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what pending from our side.

**Comment by droshev** [comment]
We need to set `aria-label="Navigate Back" title="Navigate Back" `

**Comment by JyotiKumari1507** [comment]
@droshev  please let me know which NGX version will have this fix

**Comment by JyotiKumari1507** [comment]
@droshev waiting for response

**Comment by JyotiKumari1507** [comment]
> We need to set `aria-label="Navigate Back" title="Navigate Back" `

@droshev as its ngx inbuilt code so we can't set these attribute from our side. Its should be added by your team so requesting to re-open this ticket

**Comment by g-cheishvili** [comment]
@droshev 
[this](https://github.com/SAP/fundamental-ngx/blob/main/libs/platform/table/components/table-view-settings-dialog/filtering/filter-step.component.html#L7) should be replaced with `[ariaLabel]="'platformTable.filterDialogBackToColumns' | fdTranslate"` because of [this](https://github.com/SAP/fundamental-ngx/blob/721fc49e054142194136e9657545497bf047a0ba/libs/core/button/button.component.ts#L41)

**Comment by JyotiKumari1507** [comment]
@g-cheishvili  @droshev can you please re-open this ticket, we have been asking to re-open from long time

**Comment by droshev** [comment]
you can report a new ticket with more details if needed
